src/views/Incident.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <center-wrapper>
+      <btn-back :to="{ name: 'Root' }" title="Back to the list" />
```

Comment:
Nice, I thought this was going to break the home -> filter by city -> click incident -> return to filtered by city interaction. But it still works. But... how?? :)

Where is the city stored while the incident is being shown?

Reply:
It shouldn't work like that, the city is not stored at all. It might be a caching issue? I removed the `router.go(-1)` functionality, that resulted in weird navigation when browsing through related videos.

To go back to the list of incidents by cities doesn't make any sense now we're also showing related videos on the detailpage.

Reply:
That's annoying, cmd-R fixed it, so I guess it was a caching issue. Seems like a netlify bug.

> To go back to the list of incidents by cities doesn't make any sense now we're also showing related videos on the detailpage.

I disagree, but it's a minor disagreement so I'm happy to just ship it